Lesson16/bilge/script.js

```diff
@@ -0,0 +1,416 @@
+    return 'You are old enough to vote.'
+  }
+  else if (age >= 18 && age <=65 && student === false) {
+    return 'You are eligible to vote.'
+  }
+  else {
+    return 'You are a senior citizen.'
+  }
+}
+
+console.log('Ex. 10:');
+
+console.log(checkIfStudentAndVoteEligibility(17, true));
+console.log(checkIfStudentAndVoteEligibility(18, true));
+console.log(checkIfStudentAndVoteEligibility(22, false));
+console.log(checkIfStudentAndVoteEligibility(70, false));
+
+// 11. Write a function that asks the user to enter their salary and their years of experience. If the salary is less than $30,000 and the years of experience are less than 5, print "Sorry, we cannot offer you the job at this time". If the salary is less than $50,000 and the years of experience are between 5 and 10 (inclusive), print "We can offer you the job at a lower salary". If the salary is greater than or equal to $50,000 or the years of experience are greater than or equal to 10, print "We can offer you the job at the requested salary".
+
+function decideTheSalaryOnExperience (salary, yearsOfExperience) {
+  if (salary < 30.000 && yearsOfExperience <5) {
+    return 'Sorry, we cannot offer you the job at this time.'
+  }
+  else if (salary < 50.000 && 5 <= yearsOfExperience <= 10) {
+    return 'We can offer you the job at a lower salary.'
+  }
+  else {
+    return 'We can offer you the job at the requested salary.'
+  }
+}
+
+console.log('Ex. 11:');
+
+console.log(decideTheSalaryOnExperience(25.000, 4));
+console.log(decideTheSalaryOnExperience(45.000, 10));
+console.log(decideTheSalaryOnExperience(50.000, 13));
+
+// 12. Write a function that asks the user to enter their score on a test. If the score is less than 60, print "You failed the test". If the score is between 60 and 70 (inclusive), print "You passed the test, but your grade is not great". If the score is between 71 and 80 (inclusive), print "You got a B". If the score is between 81 and 90 (inclusive), print "You got an A". If the score is greater than 90, print "You got an A+ - Great job!".
+
+function decideTheLetterGrade (score) {
+  if (score < 60) {
+    return 'You failed the test.'
+  }
+  else if (score >= 60 && score <= 70) {
+    return 'You passed the test, but your grade is not great.'
+  }
+  else if (score >= 71 && score <= 80) {
+    return 'You got a B.'
+  }
+  else if (score >= 81 && score <= 90) {
+    return 'You got a A.'
+  }
+  else {
+    return 'You got an A+ - Great job!'
+  }
+}
+
+console.log('Ex. 12:');
+
+console.log(decideTheLetterGrade(59));
+console.log(decideTheLetterGrade(60));
+console.log(decideTheLetterGrade(80));
+console.log(decideTheLetterGrade(90));
+console.log(decideTheLetterGrade(91));
+
+// 13. Write a function that asks the user to enter a month number (1-12) and displays the corresponding season name. Use a switch statement to handle the different cases.
+
+const month = new Date().getMonth();   
+const seasonName = findSeason(month);
```

Comment:
I think this code should give you an error since you try to call the function before its declaration. It would be safer to move this line below the `findSeason` function.